packages/ebayui-core/src/components/ebay-filter-input/filter-input.stories.ts

```diff
@@ -0,0 +1,176 @@
+import { buildExtensionTemplate } from "../../common/storybook/utils";
+import Readme from "./README.md";
+import Component from "./index.marko";
+import DefaultTemplate from "./examples/default.marko";
+import DefaultTemplateCode from "./examples/default.marko?raw";
+import ControlsTemplate from "./examples/controls.marko";
+import ControlsTemplateCode from "./examples/controls.marko?raw";
+
+
+
+export default {
+    title: "form input/ebay-filter-input",
+    component: Component,
+    parameters: {
+        docs: {
+            description: {
+                component: Readme,
+            },
+        },
+    },
+
+    argTypes: {
+        size: {
+            options: ["regular", "small", "large"],
```

Comment:
Type has `small` and `large` only, do we need `regular` or do we need to update the type?

Reply:
No need, regular is just the default. If set to regular then it doesn't get set.